examples/query-actions/query_actions.bal

```diff
@@ -0,0 +1,22 @@
+import ballerina/io;
+
+public function main() returns error? {
+    int[] nums = [1, 2, 3, 4];
+    int[] numsTimes10 = [];
+
+    // The `from` clause works similar to a `foreach` statement.
+    check from var i in nums
+        // The `do` statement block is evaluated for each iteration.
+        do {
+            numsTimes10.push(i * 10);
+        };
+
+    io:println(numsTimes10);
+
+    // Print only the even numbers in `nums` array.
+    check from var i in nums
+        where i % 2 == 0
```

Comment:
Shall we add a comment saying other clauses are supported here?

Reply:
I added that already in md file. Do we need to add it here again?